app/services/controlled_work_document_value_mapping_service.rb

```diff
@@ -25,7 +25,7 @@ def call(session_data, mappings)
     def convert(mapping, content)
       case mapping[:type]
       when "checkbox"
```

Comment:
so yeah, as per above, I'd change this to `always_checked_checkbox` or something now that its behaviour has changed.